Library/Formula/snownews.rb

```diff
@@ -3,16 +3,16 @@ class Snownews < Formula
   homepage "https://kiza.eu/software/snownews"
   url "https://kiza.eu/media/software/snownews/snownews-1.5.12.tar.gz"
   sha256 "26dd96e9345d9cbc1c0c9470417080dd0c3eb31e7ea944f78f3302d7060ecb90"
+  revision 1
 
   option "without-nls", "Build without translations"
 
   depends_on "gettext" if build.with? "nls"
+  depends_on "openssl"
 
-  # Fix zlib linking issue on OS X
-  # snownews author assisted on quest creating this working Formula.
-  # Author is aware of the issue tackled. However, no statement has been made whether
-  # any future release will change to a more (homebrew) robust = cleaner = simpler basis.
-  # homebrew reference added on 2013-07-06 to https://kiza.eu/software/snownews/downloading
+  # Fix system openssl linking error on OS X
+  # Allow EXTRA_LDFLAGS to take precedence so we can link brewed openssl
+  # instead of deprecated system openssl
```

Comment:
I reported it upstream by emailing the author, per [its web page](https://kiza.eu/software/snownews/faq#Feedback_and_bug_reporting).